src/resources/Sources/SourcesMappings/SourcesMappingsInterfaces.ts

```diff
@@ -0,0 +1,43 @@
+import {FieldTypes, MappingModelKind} from '../../Enums';
+
+export interface GetSourceMappingParams {
+    organizationId?: string;
+}
+
+export interface MappingsConfiguration {
+    common: {
+        rules: MappingRule[];
+    };
+    types: Array<{
+        rules: MappingRule;
+        type: string;
+    }>;
+}
+
+export interface UpdateSourceMappingModel extends GetSourceMappingParams {
+    mappingsConfiguration: MappingsConfiguration;
+}
+
+export interface MappingMappingModel {
```

Comment:
Same here, where is this used?

Reply:
This ended up not being used and has been removed :+1: